src/soc/sifive/fu540/src/ddrregs.rs

```diff
@@ -0,0 +1,173 @@
+/* Copyright (c) 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+/* See the file LICENSE for further information */
+#![no_std]
+
```

Comment:
ddrregs and phy are two separate modules, so I think you have to import phy here with something like:

import create::phy::*;